docs/security/trust/deploying_notary.md

```diff
@@ -0,0 +1,37 @@
+<!--[metadata]>
```

Comment:
<!--[metadata]>
+++
title = "Deploy Notary with Compose"
description = "Deploying Notary"
keywords = ["trust, security, notary, deployment, compose"]
[menu.main]
parent= "smn_content_trust"
+++
<![end-metadata]-->

# Deploy Notary with Compose

The easiest way to deploy Notary is by using Docker Compose. To follow the procedure on this page, you must have already [installed Compose]() Compose](http://docs.docker.com/compose/install/).

1. Clone the Notary repository

        $ git clone git@github.com:docker/notary.git

2. Build and start Notary with the sample certificates.

        $ docker-compose up -d

    For detailed documentation about how to do this at
    https://github.com/docker/notary. The sample root TLS certificate is located
    at `fixtures/root-ca.crt`. 

3. Make sure the that your Docker / Notary client trusts this certificate before you can interact with the Notary server. 

    See the instructions for [Docker](../../reference/commandline/cli.md#notary)
    and for [Notary](https://github.com/docker/notary#using-notary).

## If you want to use Notary in production

Please check back here for instructions after Notary Server has an official
stable release. To get a head start on deploying Notary in production see
https://github.com/docker/notary.